pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,102 @@
+    link: https://www.linkedin.com/in/alejandra-quetzalli/
+    byline: Our proposal? Update Docs Information Architecture
+excerpt: Check out the Docs project proposal we're submitting to GSoD 2022! You won't want to miss out.
+featured: true
+---
+
+## 🦄 Hola
+
+¡Hola, AsyncAPI community! 😄
+
+For today's blog post about AsyncAPI Docs 📑, I wanted to share with all technical writers about our organization’s plan for participation in `Google Season of Docs 2022 (GSoD)`. 
+
+As some of you may remember from my [Gist Docs update for 31 Jan - 11 Feb 2022](https://gist.github.com/alequetzalli/94ca1ffb5d123b450501e40a4a3b56e2), I noted that GSoD 2022 was coming up and that AsyncAPI wanted to participate in the application process once it opened on February 23, 2022.  
+
+In anticipation of this, I also created a new AsyncAPI Slack channel named `#temp-gsod-2022` that anyone can join! First, [join our Slack workspace](https://www.asyncapi.com/slack-invite) ☎️  and please respect [our slack etiquette](https://github.com/asyncapi/.github/blob/master/slack-etiquette.md).🙂 Then join the `temp-gsod-2022` channel, our temporary channel to coordinate GSoC 2022 setup. I'll publish regular updates on where we are in the application process, so stay tuned as the process continues. 🙂
+
+Join the `#temp-gsod-2022` slack channel for:
+- mentees identification
+- mentors identification
+- ideas identification
+- mentees and ideas and mentors matching
+
+
+Below is the **project proposal** we're submitting to `GSoD 2022` and then we close with a reminder of how to get started as an AsyncAPI Docs contributor:
+
+
+___ 
+
+# 🙌🏾  Update Docs Information Architecture - AsyncAPI Initiative 
+
+## ❤️ About AsyncAPI
+AsyncAPI (currently version 2.3.0, first released in 2016) is an Apache License 2.0 library [under the Linux Foundation](https://www.linuxfoundation.org/press-release/linux-foundation-will-host-asyncapi-to-support-growth-and-collaboration-for-industrys-fastest-growing-api-spec/) that seeks to improve the current state of Event-Driven Architectures (EDA). The AsyncAPI Initiative is a specification and growing set of open-source tools to help developers define asynchronous APIs, and build and maintain event-driven architectures. Developers familiar with OpenAPI (aka Swagger) for RESTful APIs will see strong similarities when using AsyncAPI. One common use case is generating documentation (HTML or Markdown) of an asynchronous API. The specification is both platform and language agnostic. Current tooling includes support for common message brokers such as Apache Kafka and RabbitMQ, and languages including Python, Java, and Nodejs. Our long-term goal is to make working with EDAs as easy as working with REST APIs. That goes from documentation to code generation, from discovery to event management, and beyond. Our 150+ Open-Source (OSS) contributors are EDA enthusiasts from all around the world. 
+
+
+## 📑 About our Docs project
+
+### 🔎 Our current Docs problem
```

Comment:
Would it make sense to rename this section as `State of documentation at AsyncAPI`?

Reply:
No, because we are following the template proposal as Google dictates it :) 

https://developers.google.com/season-of-docs/docs/org-proposal-template